src/averageAges.js

```diff
@@ -38,6 +46,14 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const women = people.filter(person => person.sex === 'f');
+  const allChildren = people.map(children => children.mother);
```

Comment:
in cases when 'withChildren' is false this is unnecessary, do it only when 'withChildren' is true